### ISSUE TITLE
Avatares en Unity

### ISSUE THREAD

**Issue by cambridgekids** [cambridgekids/David-games#13]
Estoy echando un vistazo a avatares en Unity y me parece muy fuerte que podemos tenerlos en 3D.
https://assetstore.unity.com/packages/3d/characters/humanoids/toon07-102204
1. Have a look a esta niña. La tenemos que cambiar la cara eso está claro ya que es para niños.
2. Creo que su cara es una png "wrapped" around the model" tú sabes manipular todo eso verdad?
    Es decir sería abrir la png y manipularla o crear otra png con una cara más bonita y ya está.
3. PERO no veo que se abre la boca. Osea para hablar con los niños molaría que hablara tipo "Dora la exploradora".

**Comment by Deivith** [comment]
Hola Gerry

2 - Yo no se manipular las texturas. Es decir entiendo como se funcionan, pero las modificaciones que puedo realizar son minimas. Pero tengo modeladores 3D e Ilustradores con los que trabajo y saben hacer ese trabajo.

3 - Se puede animar la textura tambien si hiciera falta (se puede buscar un animador que modifique la ilustracion).

**Comment by cambridgekids** [comment]
Hola. Pues voy a mirar unas opciones y si tienes algún compañero de "fiar" y a buen precio se podría ir mirando cositas. Será mejor comprar un modelo ya montado como asset. Rigging creo que es rápido. Animación del modelo no tengo ni idea.
https://www.turbosquid.com/3d-models/cartoon-kid-3d-model/1084832
He visto este sitio pero no sé si estos modelos son demasiado high poly. Lo sabrá tus compañeros?

**Comment by Deivith** [comment]
No, estan bien. No son tan high poly. Igualmente quizas si tienes un par de personajes en mente puedo pedir presupuesto y seguro se consigue mas barato. 100 dolares ese modelo sin animaciones es caro. Puedo conseguir mas barato! Es de fiar, es la persona con la que estoy haciendo mi juego.
